blog/2023-08-17-bssw-fellowship/index.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Notes from my Better Scientific Software (BSSw) Fellowship"
+# description: |
+#  reflections of a first-time freelancer
+author:
+  - name: Julie Lowndes
+    orcid: 0000-0003-1682-3872
+date: 2023-08-17
+citation: 
+  url: https://openscapes.org/blog/2023-08-17-bssw-fellowship/
+categories: [how-we-work]
+image: openscapes-champions-series.png
+---
+
+*From Spring 2022-2023 I have had the pleasure of being a Better Scientific Software (BSSw) Fellow, by the Exascale Computing Project (17-SC-20-SC), a collaborative effort of the U.S. Department of Energy (DOE) Office of Science and the National Nuclear Security Administration; and by the National Science Foundation (NSF) under Grant No. 2154495. This is an excerpt from my final progress report to the The Krell Institute, who administered this award.*
+
+*Quick links:*
+
+-   [*How Open Source Tooling Is Changing the Way Professional Researchers Learn to Code*](https://bssw.io/blog_posts/how-open-source-tooling-is-changing-the-way-professional-researchers-learn-to-code) *- my post on the BSSw blog*
+
+------------------------------------------------------------------------
+
+My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and I have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
```

Comment:
```suggestion
My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and we have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
```